### PR TITLE
chore(api-ref): update API ref scripts for v3:

### PR DESCRIPTION
**Pull request by jstirnaman**
- Replace cloud-iox with cloud-serverless and cloud-dedicated in getswagger.sh
    - Add cloud-dedicated API ref (handcrafted) with query, write, and ping paths.
    - Remove separate v1-compat docs for Serverless and Dedicated. The v1 paths are included in contracts/ref.
    - Remove separate v1 docs buttons from api/_index.md

image
image
image

### CODE REVIEW

File: api-docs/cloud-dedicated/ref.yml [hunk 1]
```diff
@@ -0,0 +1,2060 @@
...
+              message:
+                description: A human-readable message that may contain detail about the error.
+                readOnly: true
+                type: string
+      description: |
+        Unauthorized. The error may indicate one of the following:
+
+          * The `Authorization: Token` header is missing or malformed.
+          * The API token value is missing from the header.
+          * The token doesn't have sufficient permissions to write to the database.
```
Comment (sanderson) [inline api-docs/cloud-dedicated/ref.yml:43]: Is this context specific to writing, or should this say "... sufficient permissions to write to or query the database."?

File: api-docs/cloud-dedicated/ref.yml [hunk 2]
```diff
@@ -0,0 +1,2060 @@
...
+            orgProvidedNotFound:
+              summary: The org or orgID passed doesn't own the token passed in the header
+              value:
+                code: invalid
+                message: "failed to decode request body: organization not found"
+          schema:
+            $ref: "#/components/schemas/Error"
+      description: |
+        Bad request.
+        The response body contains detail about the error.
```
Comment (sanderson) [inline api-docs/cloud-dedicated/ref.yml:57]:
```suggestion
        The response body contains details about the error.
```

**Review by sanderson — APPROVED** [review]
Looks good!

File: api-docs/cloud-dedicated/ref.yml (hunk 1 above)
Comment (jstirnaman) [inline api-docs/cloud-dedicated/ref.yml:43]:
```suggestion
          * The token doesn't have sufficient permissions to write to or query the database.
```

**Pull request merged**